### ISSUE TITLE
Quality Menu

### ISSUE THREAD

**Issue by maxens-git** [jellyfin/Swiftfin#184]
it would be nice if there was a menu to choose the quality directly from the video extent and not from the application settings
Thanks for your reading

**Comment by LePips** [comment]
I'll have this as a part of my video player refactor, but this may come long after that.

**Comment by gray19950103** [comment]
> I'll have this as a part of my video player refactor, but this may come long after that.

is there any updates on this feature? Or is there any other way I can change the quality streaming?
In my case, I need to use Apple TV and Infuse dosen't provide quality menu, with my old non4K Apple TV, some large bitrates movie cause stuttering. (my iPhone & Mac works fine)
So I guess if I could change the transcode option to lower bitrate while playing on Apple TV, maybe will solve this problem.

**Comment by teddyursa** [comment]
> > I'll have this as a part of my video player refactor, but this may come long after that.
> 
> is there any updates on this feature? Or is there any other way I can change the quality streaming with? In my case, I need to use Apple TV and Infuse dosen't provide quality menu, with my old non4K Apple TV, some large bitrates movie cause stuttering. (my iPhone & Mac works fine) So I guess if I could change the transcode option to lower bitrate while playing on Apple TV, maybe will solve this problem.

Did you have any luck with this? I'm struggling with 1080p content and it's frustrating there's not much chatter about the lack of a menu to help with buffering.

**Comment by gray19950103** [comment]
> > > I'll have this as a part of my video player refactor, but this may come long after that.
> > 
> > 
> > is there any updates on this feature? Or is there any other way I can change the quality streaming with? In my case, I need to use Apple TV and Infuse dosen't provide quality menu, with my old non4K Apple TV, some large bitrates movie cause stuttering. (my iPhone & Mac works fine) So I guess if I could change the transcode option to lower bitrate while playing on Apple TV, maybe will solve this problem.
> 
> Did you have any luck with this? I'm struggling with 1080p content and it's frustrating there's not much chatter about the lack of a menu to help with buffering.

No, it seems you can only wait for quality select menu provided by Infuse or Swiftfin someday. Now I'm temporarily using this way: play with infuse on ios and airplay to Apple TV, but sometimes you can't lock the screen of ios device.

**Comment by isvein** [comment]
Pops in to say I miss this too.

**Comment by matthias-hoste** [comment]
Is it possible to give bounties for issues? Because I'd really like to see this feature realized

**Comment by thornbill** [comment]
No bounties.

**Comment by 100tomer** [comment]
+1 for this feature request

**Comment by oleggtro** [comment]
Just wanted to pop in to ask whether someone has already begun on this topic. I don't know just how hard this is to implement, but i certainly want to try it

**Comment by 100tomer** [comment]
> Just wanted to pop in to ask whether someone has already begun on this topic. I don't know just how hard this is to implement, but i certainly want to try it

I wish I knew how to do that, but I’m glad to see that you might.

**Comment by albarago** [comment]
Hello, has there been any traction with this one?

**Comment by JPKribs** [comment]
> Hello, has there been any traction with this one?

https://github.com/jellyfin/Swiftfin/pull/1581
